browser/ui/views/sidebar/sidebar_show_options_event_detect_widget.cc

```diff
@@ -115,5 +115,11 @@ void SidebarShowOptionsEventDetectWidget::AdjustWidgetBounds() {
   rect.set_width(kWidgetNarrowWidth);
 
   contents_view_->SetPreferredSize(rect.size());
+
+#if BUILDFLAG(IS_MAC)
+  // On Mac, we can set empty bounds for the widget.
+  if (rect.IsEmpty())
+    rect.set_size({kWidgetNarrowWidth, 1});
```

Comment:
Hmm.. I think `rect` could not be empty because `rect` is from contents container.
I'm fine this for safe but just curious it could be happened on macOS.

Reply:
Yeah, for safety. I reached here when there's no height limit.